src/AliceInventory.IntegrationTests/PostTest.cs

```diff
@@ -0,0 +1,112 @@
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
+[assembly: TestCaseOrderer("Xunit.Extensions.Ordering.TestCaseOrderer", "Xunit.Extensions.Ordering")]
+[assembly: TestCollectionOrderer("Xunit.Extensions.Ordering.CollectionOrderer", "Xunit.Extensions.Ordering")]
+namespace AliceInventory.IntegrationTests
+{
+    [Order(1)]
+    public class PostTest
+    {
+        private readonly TestServer _server;
+        private readonly HttpClient _client;
+
+        public PostTest()
+        {
+            _server = new TestServer(WebHost.CreateDefaultBuilder().UseStartup<AliceInventory.Startup>());
+            _client = _server.CreateClient();
+        }
+
+        //private string jsonExample = "{\"meta\":{\"locale\":\"ru-RU\",\"timezone\":\"Europe/Moscow\",\"client_id\": \"ru.yandex.searchplugin/5.80 (Samsung Galaxy; Android 4.4)\",    \"interfaces\": {      \"screen\": { }    }  },  \"request\": {    \"command\": \"*TEXT*\",    \"original_utterance\": \"������ ����� �� ����� ���� ��������, 16 �� ������\",    \"type\": \"SimpleUtterance\",    \"markup\": {      \"dangerous_context\": true    },    \"payload\": {},    \"nlu\": {      \"tokens\": [        \"������\",        \"�����\",        \"��\",        \"����\",        \"��������\",        \"16\",        \"��\",        \"������\"      ],      \"entities\": [        {          \"tokens\": {            \"start\": 2,            \"end\": 6          },          \"type\": \"YANDEX.GEO\",          \"value\": {            \"house_number\": \"16\",            \"street\": \"���� ��������\"          }        },        {          \"tokens\": {            \"start\": 3,            \"end\": 5          },          \"type\": \"YANDEX.FIO\",          \"value\": {            \"first_name\": \"���\",            \"last_name\": \"�������\"          }        },        {          \"tokens\": {            \"start\": 5,            \"end\": 6          },          \"type\": \"YANDEX.NUMBER\",          \"value\": 16        },        {          \"tokens\": {            \"start\": 6,            \"end\": 8          },          \"type\": \"YANDEX.DATETIME\",          \"value\": {            \"day\": 1,            \"day_is_relative\": true          }        }      ]    }  },  \"session\": {    \"new\": true,    \"message_id\": 4,    \"session_id\": \"2eac4854-fce721f3-b845abba-20d60\",    \"skill_id\": \"3ad36498-f5rd-4079-a14b-788652932056\",    \"user_id\": \"AC9WC3DF6FCE052E45A4566A48E6B7193774B84814CE49A922E163B8B29881DC\"  },  \"version\": \"1.0\"}";
+
+        private StringContent CreateJsonContent(string text)
+        {
+            AliceRequest response = new AliceRequest() { Request = new RequestModel() { Command = text }};
+            string json = JsonConvert.SerializeObject(response);
+            return new StringContent(json, Encoding.Default, "application/json");
+        }
+        
+        [Theory, Order(1)]
+        [InlineData("������ 3 ������", new []{ "������ ����� 3 ��" })]
```

Comment:
What type of charset you have used? I suppose, that it is not the UTF-8, therefore I have problems to see correct text in Russian inside VS Code and GitHub
![image](https://user-images.githubusercontent.com/32573127/57979080-2ed91980-7a21-11e9-8d96-a29e7553601a.png)

Reply:
Already [fixed](https://github.com/vkamiansky/hackathonfls2019/commit/3bbd0963957d605cba5661a004e774b18f03db88)